### DISCUSSION TITLE
RP2040 CDC MSC

### DISCUSSION THREAD

**Discussion by kaboffienc** [hathach/tinyusb#1881] [General]
Hello all,
I am working with a project where I plan to use rp2040 tiny usb device to store some files >5MB in a SDCARD.
My question is any ideas on how to make the MSC device to point to the address or path generate by rp2040 to the  SDCARD mounted ??. and if not possible to change the pointing address will be good to know too.
Thanks.

**Comment by hathach** [comment]
block address is already exposed by tinyusb API. check out the cdc_msc example for detail
